internal/ci/vulnerability_result_diff.go

```diff
@@ -65,3 +65,32 @@ func DiffVulnerabilityResults(oldRes, newRes models.VulnerabilityResults) models
 
 	return result
 }
+
+// DiffVulnerabilityResultsByUniqueVulnCount will return a count of unique vulnerabilities that are in `newRes`
```

Comment:
Nit: can we better define "unique" in the comment? It seems a bit confusing to me as is.

Alternatively, we can describe this in terms of "occurrences", which seems closer to what we're actually doing. 

e.g. "DiffVulnResultsByOccurences". "Return a count of new occurrences of vulnerabilities, while ignoring the exact source the vulenrability was found in". 

Also probably worth adding a comment here on the use case this serves: "this prevents us reporting "new" vulnerabilities in a PR when a previously vulnerable file is being moved."